examples/raw_reference_cast.cu

```diff
@@ -100,7 +100,7 @@ int main(void)
 
   // note: we must specify the System to ensure correct execution
   thrust::for_each(thrust::counting_iterator<int,System>(0),
-                   thrust::counting_iterator<int,System>(N),
+                   thrust::counting_iterator<int,System>(static_cast<int>(N)),
```

Comment:
Because this is in an example, I would just make `N` be `int` or change the `counting_iterator` type to be `size_t`, so we can avoid a `static_cast` in example code.

Reply:
Done.